packages/indexer/indexer/chain_mapper.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass, field
+from typing import Dict, List
+
+
+@dataclass
+class CosmosChain:
+    chain_id: str
+    min_block_height: int
+    apis: List["CosmosAPI"] = field(default_factory=list)
+    current_api_index: int = 0
+
+
+@dataclass
+class CosmosAPI:
+    url: str
+    archive: bool = False
+
+
+chain_mapping: List[CosmosChain] = [
+    CosmosChain(
+        chain_id="secret-4",
+        min_block_height=7284419,
+        apis=[
+            CosmosAPI(url="https://secret-4.api.trivium.network:1317"),
```

Comment:
Same thing here - no hardcoding of endpoints

Reply:
agreed should all be in a `.env`